src/package.ts

```diff
@@ -53,9 +57,9 @@ export class Package extends AsyncOptionalCreatable {
     this.location = location;
   }
 
-  public async readProjectJson(): Promise<ProjectJson> {
+  public async readPackageJson(): Promise<PackageJson> {
     const pkgJsonPath = this.location ? path.join(this.location, 'package.json') : 'package.json';
-    return (await fs.readJson(pkgJsonPath)) as ProjectJson;
+    return (await fs.readJson(pkgJsonPath)) as PackageJson;
```

Comment:
This is essentially the same as the exported function above.

1. should we reuse 1 function?
2. should we just make it a synchronous read (readJsonSync) or is async more appropriate?